### ISSUE TITLE
From local dev to CI environment : What to do with the file content?

### ISSUE THREAD

**Issue by dolanor** [dagger/dagger#1845]
Visiting https://docs.dagger.io/1201/ci-environment, we see a yaml file for github actions.
What are we supposed to do at this part of the documentation? It's copyable.
We just got the local `dagger do build` work. Are we supposed to do the next step on github actions directly here?

From what I understand, it's just a way how to integrate with other CI, but this is just documentation, not part of the tutorial we did right before. It confused me.

**Comment by samalba** [comment]
@dolanor (or anyone) do you have an idea how to address this one?

I would indeed add a quick paragraph for each CI that points to the corresponding CI doc (e.g. here is how to create a github actions workflow with github, move the file to .github/workflows/todoapp.cue at the root of the repository, etc...)

**Comment by dolanor** [comment]
I guess the suggestion for quick CI introduction is a good idea.
What confused me in the first place is that the example was deep inside of universe, and putting/creating the file in the `.github` dir inside the dagger repo would be confusing, as there is other GHA config files.
Now that we have a dedicated todoapp repo, it is less confusing to add this file to your own fork of the todoapp.
So I guess, once the todoapp documentation is updated, with links pointing to the new repo, we could update this part as well and exactly tell what we want to do with it.

**Comment by samalba** [comment]
> So I guess, once the todoapp documentation is updated, with links pointing to the new repo, we could update this part as well and exactly tell what we want to do with it.

Is it this one #2484 ? It just got merged in case it unblocks you for this one.

**Comment by mircubed** [comment]
@dolanor just wanted to check-in to see if this was still blocked?

**Comment by dolanor** [comment]
I tried it.
I cloned the dagger/todoapp repo, I copy/pasted the content of the workflows, and it failed with:
```
  1:35PMINFclient.filesystem."./build".write | completed    duration=200ms
  1:35PMINFactions.deploy.container._exec | #19 0.364 cat: can't open '/run/secrets/token': No such file or directory

```
It was because I didn't set a `NETLIFY_TOKEN`. But it wasn't explicit about it.

I set the secret and it blocked for another issue:
```
  1:43PMINFactions.deploy.container._exec | #19 10.92  ›   JSONHTTPError: Unprocessable Entity
  1:43PMINFactions.deploy.container._exec | #19 10.94 Error creating site [runner-dagger-todoapp] for account []
  1:43PMERRactions.deploy.container._exec | failed: process "bash --norc -e -o pipefail /bash/scripts/deploy.sh" did not complete successfully: exit code: 1
```
I thought because my site didn't have this name, so I set it up with this name, but it didn't fix it.
I don't know much about netlify, so if it rings somebody's bell, that could be nice
cc: @shykes

**Comment by dolanor** [comment]
It was a wrong naming of my netlify app from previous tests.
So now it's working, but again, the discoverability of what was the issue is bad.

**Comment by dolanor** [comment]
Trying this part of the tutorial once again to see if the issue is fixed.

**Comment by dolanor** [comment]
This tutorial is not directly accessible anymore (though the URL link still works).
